sensor/kubernetes/complianceoperator/updater_test.go

```diff
@@ -100,6 +100,61 @@ func (s *UpdaterTestSuite) TestDelayedTicker() {
 	}, actual)
 }
 
+// mockRequiredResources creates a list of mock required resources for testing.
+func mockRequiredResources() []metaV1.APIResource {
+	// loop through the list of required resources and return a list of APIResources
+	var kinds []string
+	for _, resource := range complianceoperator.GetRequiredResources() {
+		kinds = append(kinds, resource.Kind)
+	}
+	return convertToAPIResourceList(kinds)
+
+}
+
+// convertToAPIResourceList converts a string slice to an APIResourceList.
+func convertToAPIResourceList(kinds []string) []metaV1.APIResource {
+	var resources []metaV1.APIResource
+	for _, kind := range kinds {
+		resources = append(resources, metaV1.APIResource{Kind: kind})
+	}
+	return resources
+}
+
+func (s *UpdaterTestSuite) TestCheckRequiredComplianceCRDsExist() {
+	// Setup
+	requiredResources := mockRequiredResources()
+	detectedKinds := make(map[string]bool)
+
+	for _, resource := range requiredResources {
+		detectedKinds[resource.Kind] = true
+	}
+
+	// Test case where all required CRDs exist
+	err := checkRequiredComplianceCRDsExist(&metaV1.APIResourceList{APIResources: requiredResources})
+	// assert.NoError(t, err, "checkRequiredComplianceCRDsExist should return no error when all required CRDs are present")
```

Comment:
Could you remove this commented out code, please? Maybe transfer the failure description to line 135, so we get a nice explanation why the test fails

Reply:
thanks for the review, just updated the code